tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist = py27
+
+[testenv]
+usedevelop = true
+deps =
+    -rrequirements.txt
+    pyftpdlib
+    pytest-xdist
+commands =
+    py.test {posargs:tests}
```

Comment:
agreeing with everything. didn't know about setup.py, that's useful to know.

so the only thing left here is to integrate tox into travis, IMHO, plus the task you added to fix extra tests.

thanks!